bioblend/galaxy/invocations/__init__.py

```diff
@@ -128,6 +131,89 @@ def show_invocation(self, invocation_id):
         url = self._make_url(invocation_id)
         return self._get(url=url)
 
+    def rerun_invocation(self, invocation_id: str, inputs_update: Optional[dict] = None,
+                         params_update: Optional[dict] = None, history_id: Optional[str] = None,
+                         history_name: Optional[str] = None, import_inputs_to_history: bool = False,
+                         replacement_params: Optional[dict] = None, allow_tool_state_corrections: bool = False,
+                         inputs_by: Optional[str] = None, parameters_normalized: bool = False):
+        """
+        Rerun a workflow invocation.
+
+        :type invocation_id: str
+        :param invocation_id: Encoded workflow invocation ID to be rerun
+
+        :type history_id: str
+        :param history_id: The encoded history ID where to store the workflow
+          output. Alternatively, ``history_name`` may be specified to create a
+          new history.
+
+        :type history_name: str
+        :param history_name: Create a new history with the given name to store
+          the workflow output. If both ``history_id`` and ``history_name`` are
+          provided, ``history_name`` is ignored. If neither is specified, a new
+          'Unnamed history' is created.
+
+        :type import_inputs_to_history: bool
+        :param import_inputs_to_history: If ``True``, used workflow inputs will
+          be imported into the history. If ``False``, only workflow outputs will
+          be visible in the given history.
+
+        :type allow_tool_state_corrections: bool
+        :param allow_tool_state_corrections: If True, allow Galaxy to fill in
+          missing tool state when running workflows. This may be useful for
+          workflows using tools that have changed over time or for workflows
+          built outside of Galaxy with only a subset of inputs defined.
+
+        :type replacement_params: dict
+        :param replacement_params: pattern-based replacements for post-job
+          actions
+
+        :type inputs_by: str
+        :param inputs_by: Determines how inputs are referenced. Can be
+          "step_index|step_uuid" (default), "step_index", "step_id", "step_uuid", or "name".
+
+        :type parameters_normalized: bool
+        :param parameters_normalized: Whether Galaxy should normalize ``params``
+          to ensure everything is referenced by a numeric step ID. Default is
+          ``False``, but when setting ``params`` for a subworkflow, ``True`` is
```

Comment:
``params`` is not an argument of this method (it is of `WorkflowClient.invoke_workflow()`), so this help should be revised.